src/pybind/mgr/zabbix/module.py

```diff
@@ -88,12 +88,8 @@ def __init__(self, *args, **kwargs):
     def init_module_config(self):
         for key, default in self.config_keys.items():
             value = self.get_localized_config(key, default)
-            if value is None:
-                raise RuntimeError('Configuration key {0} not set; "ceph '
-                                   'config-key set mgr/zabbix/{0} '
-                                   '<value>"'.format(key))
-
-            self.set_config_option(key, value)
+            if value is not None:
+                self.set_config_option(key, value)
```

Comment:
i don't understand the rational behind this change. before this change: the options are always applied as long as they are not `None`. and we bail out when any of them is `None`. 

with your change, we are more to tolerant: we move on even if an option is None.

i think we **cannot** continue if some option is not configured beforehand, for example, we cannot guess a `zabbix_host` for user.

Reply:
When some key value is None,we will panic here,then the default value for  zabbix_port/zabbix_sender/interval will not be set.

First,start the mgr program with nothing to set,then i set the zabbix_host and identifier which must be set to use the zabbix plugin.
Then if run ceph zabbix send ,In the zabbixsender init will be supposed to think the other three
default key must have been set,but actually those key does not set.
So finally  the ceph zabbix send will be failed.

Reply:
The point is that those three key which should have default value should set by the set_config_option even some other key have None value.